include/librealsense2/h/rs_option.h

```diff
@@ -81,6 +81,7 @@ extern "C" {
         RS2_OPTION_ENABLE_POSE_JUMPING, /**< Enable position jumping */
         RS2_OPTION_ENABLE_DYNAMIC_CALIBRATION, /**< Enable dynamic calibration */
         RS2_OPTION_DEPTH_OFFSET, /**< Offset from sensor to depth origin in millimetrers*/
+        RS2_OPTION_ENABLE_MAP_PRESERVATION, /**< Preserve previous map when starting */
```

Comment:
Pls keep the original order and append to end

Reply:
Thanks for pointing this out.  This needs a rebase after 3f82376.